WebDriverAgentLib/Categories/XCUIElement+FBUID.m

```diff
@@ -19,7 +20,11 @@ - (NSString *)fb_uid
   if ([self respondsToSelector:@selector(accessibilityElement)]) {
     return [FBElementUtils uidWithAccessibilityElement:[self performSelector:@selector(accessibilityElement)]];
   }
-  return self.fb_lastSnapshot.fb_uid;
+  if (isSDKVersionLessThan(@"13.0")) {
```

Comment:
I'd rather wrap `[self fb_nativeResolve]` call with `if`, since we anyway have to return the same thing.
Also, it makes sense to add an explanation comment here